### PR TITLE
GetFriendlyUrl must respect page external url

### PR DESCRIPTION
**Pull request by gatisb**
see: https://github.com/Geta/EPi.Extensions/issues/40

### CODE REVIEW

File: src/Geta.EPi.Extensions/ContentReferenceExtensions.cs [hunk 1]
```diff
@@ -135,7 +135,15 @@ public static string GetFriendlyUrl(this ContentReference contentReference, stri
...
                 : urlResolver.GetUrl(contentReference, language);
 
-            return includeHost ? url.AddHost() : url.RemoveHost();
+            if (includeHost)
+            {
+                return url.AddHost();
+            }
+
+            var content = Get<IContent>(contentReference);
+            var pageData = content as PageData;
```
Comment (marisks) [inline src/Geta.EPi.Extensions/ContentReferenceExtensions.cs]: Why page data? What if it is a media content or other content?

Comment (gatisb) [reply src/Geta.EPi.Extensions/ContentReferenceExtensions.cs]: then its null, and there is no external url

Comment (marisks) [reply src/Geta.EPi.Extensions/ContentReferenceExtensions.cs]: So only page data can have external links?

Comment (gatisb) [reply src/Geta.EPi.Extensions/ContentReferenceExtensions.cs]: yes

**Pull request merged**